### PR TITLE
Allows to set a custom Relay implementation

### PR DESCRIPTION
**Pull request by tdraier**

### CODE REVIEW

File: src/main/java/graphql/annotations/GraphQLAnnotations.java [hunk 1]
```diff
@@ -571,7 +574,7 @@ protected GraphQLFieldDefinition getField(Method method) throws GraphQLAnnotatio
             List<GraphQLFieldDefinition> fieldDefinitions = outputType instanceof GraphQLObjectType ?
                     ((GraphQLObjectType) outputType).getFieldDefinitions() :
                     ((GraphQLInterfaceType) outputType).getFieldDefinitions();
-            relay = RELAY_TYPES.mutationWithClientMutationId(title, method.getName(),
+            relay = GraphQLAnnotations.this.relay.mutationWithClientMutationId(title, method.getName(),
```
Comment (guy120494) [inline src/main/java/graphql/annotations/GraphQLAnnotations.java]: Please remove the ``GraphQLAnnotations.this``

Comment (tdraier) [reply src/main/java/graphql/annotations/GraphQLAnnotations.java]: done

**Review by yarinvak — APPROVED** [review]
looks good :)

**Comment by guy120494** [comment]
Please update the readme file in the right section, and I will merge it

**Pull request merged**